openapi/internal/jsoac/maximum.go

```diff
@@ -0,0 +1,12 @@
+package jsoac
+
+import (
+	schema "github.com/jsightapi/jsight-schema-core"
+)
+
+func newMaximum(astNode schema.ASTNode, t OADType) *Example {
```

Comment:
To have fewer questions, I would make a new **Number** structure. **Number** is a truncated version of **Example**.

It is better to return a reference to the **Number** structure from this function